test/integration/common/StudioActions.kt

```diff
@@ -72,12 +73,34 @@ object StudioActions {
         clickAllInstancesOfText(composeRule, icon.unicode)
     }
 
-    fun assertNodeExistsWithText(composeRule: ComposeContentTestRule, text: String): SemanticsNodeInteraction {
-        return composeRule.onNodeWithText(text).assertExists()
+    suspend fun waitUntilNodeWithIconIsClickable(composeRule: ComposeContentTestRule, icon: Icon) {
```

Comment:
Totally agree with the new strategy of waiting for conditions rather than expecting them to be true immediately. But I don't like the fact that we've lost the word `assert` from the method names. Tests generally read better when it's clear what they're testing.